components/content-picker/SortableList.tsx

```diff
@@ -19,6 +33,35 @@ interface SortableListProps {
 	setPosts: (posts: Array<PickedItemType>) => void;
 }
 
+type Term = {
+	count: number;
+	description: string;
+	id: number;
+	link: string;
+	meta: { [key: string]: unknown };
```

Comment:
This is an opinionated suggestion and purely stylistic. Feel free to ignore if this is like this elsewhere on the project or if you don't like it either :) . Many developers (myself included) prefer using Record for its brevity and clarity, especially when defining mappings between keys and values.

```suggestion
	meta: Record<string, unknown>;
```